Assets/Scripts/Services/Input/InputController.cs

```diff
@@ -0,0 +1,66 @@
+using System;
+using UnityEngine;
+using UnityEngine.InputSystem;
+using Zenject;
+
+namespace Services.Input
+{
+    public class InputController : IInputController
+    {
+        private readonly InputActionAsset _inputActionAsset;
+        
+        public event Action<Vector2> MoveChanged;
+        public event Action<Vector2> FireChanged;
+        public event Action<Vector2> SpecialChanged;
+        public event Action MeleeChanged;
+        public event Action UseChanged;
+        public event Action ReloadChanged;
+
+        [Inject]
+        public InputController(InputActionAsset inputActionAsset)
+        {
+            _inputActionAsset = inputActionAsset;
+            OnEnable();
+        }
+
+        private void Move(InputAction.CallbackContext ctx)
```

Comment:
Rename Method to OnMove. Also pls rename all another handlers in this class

Reply:
Done